torch/csrc/jit/python_ir.cpp

```diff
@@ -391,7 +391,7 @@ void initPythonIRBindings(PyObject * module_) {
         for (size_t i = 0; i < v.size(); ++ i) {
             v[i] = autograd::Variable(v[i].view({})).data();
         }
-        return n.ts_(Symbol::attr(name), std::move(v));
+        return n.ts_(Symbol::attr(name), v);
```

Comment:
Actually, I think you do want the move here. `v` is some sort of vector-y thing, right? So it can be profitably moved. (And I'm pretty sure `ts_` takes a `std::vector`, not an ArrayRef)

Reply:
ya, I think this should change.